From: Jiri Slaby <jslaby@suse.cz>
Date: Sat, 11 Mar 2017 09:13:49 +0100
Subject: proper compiler flags check
Patch-mainline: no

Check the compiler flags with -Werror so that the output is not flooded
with unsupported options.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 Rules.mak |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Rules.mak
+++ b/Rules.mak
@@ -127,7 +127,7 @@ endif
 
 # A nifty macro to make testing gcc features easier
 check_gcc=$(shell \
-	if $(CC) $(1) -S -o /dev/null -xc /dev/null > /dev/null 2>&1; \
+	if $(CC) -Werror $(1) -S -o /dev/null -xc /dev/null > /dev/null 2>&1; \
 	then echo "$(1)"; else echo "$(2)"; fi)
 check_as=$(shell \
 	if $(CC) -Wa,$(1) -Wa,-Z -c -o /dev/null -xassembler /dev/null > /dev/null 2>&1; \
